.github/ISSUE_TEMPLATE/release.md

```diff
@@ -45,9 +45,10 @@ The following checks can be performed after we have frozen our release candidate
     https://github.com/Manta-Network/Manta/releases with relevant [release
     notes](#release-notes)
 - [ ] Check that build artifacts have been added to the
-    draft-release
+    draft-release 
 - [ ] Coordinate with marketing team for documentation updates and other relevant tasks.
 - [ ] Update changelog.
+- [ ] If the release contains any changes that break/change functionality used in https://github.com/Manta-Network/sdk (e.g. RPC changes, see also [extrinsic ordering](#extrinsic-ordering)), raise a PR there and **block this release** until your PR has been merged and incorporated in a new SDK release.
 
 Note: Do not publish draft releases from PR branches, because those branches will be deleted when the PR is merged.
 
```

Comment:
Could you also change the references of `Como` to be `Rococo`. And place them after references to Baikal. 
We retired Como and upgrade Baikal first now.